<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign_in</class>
 <widget class="QMainWindow" name="sign_in">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1311</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/playground.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_skull_pirate">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/Pirate skull.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_skull_pirate_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/Pirate skull.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_userrname">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Username :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_password">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Password : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_username">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>120</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_password">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>310</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_log_in">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>450</y>
      <width>321</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>50</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/playground.png);</string>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_forgot">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>450</y>
      <width>321</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/playground.png);</string>
    </property>
    <property name="text">
     <string>Forgot password ?</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_skull_king_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>321</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/back-of-card.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tempus Sans ITC</family>
      <pointsize>50</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/photos/playground.png);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
